lib/generator/implementation.rb

```diff
@@ -35,6 +35,8 @@ class ImplementationDelegator < SimpleDelegator
   # A implementation that also logs its progress.
   class LoggingImplementation < ImplementationDelegator
     def initialize(implementation:, logger:)
+      super
```

Comment:
This may not even be necessary.  But was not incorrect to use the "no pass through" directive here that is indicated with the `super()`, a quick check indicates the arguments are used locally, and not intended for consumption further up the ancestor chain.  The same quick looking suggests that we would not even need to call the parents `initialize` method.

Reply:
Disregard the above for now.  Found the reason for it.